spring-boot-project/spring-boot-testcontainers/src/main/java/org/springframework/boot/testcontainers/lifecycle/TestcontainersLifecycleBeanPostProcessor.java

```diff
@@ -56,7 +56,7 @@ class TestcontainersLifecycleBeanPostProcessor implements DestructionAwareBeanPo
 
 	private static final Log logger = LogFactory.getLog(TestcontainersLifecycleBeanPostProcessor.class);
 
-	private ConfigurableListableBeanFactory beanFactory;
+	private final ConfigurableListableBeanFactory beanFactory;
```

Comment:
Why is this better as non-final?

Reply:
@jonatan-ivanov I don't know its exact performance implication, but I thought expressing its immutability at least helps its readability.

Reply:
I think I'm tired, I could swear that I read that `final` got removed here instead of added, it makes sense, sorry for the confusion.